test/e2e/test/version.go

```diff
@@ -6,19 +6,27 @@ package test
 
 import (
 	"testing"
+)
 
-	"github.com/stretchr/testify/require"
-	"k8s.io/apimachinery/pkg/util/version"
+// Elastic Stack versions used in the E2E tests
+var (
+	// Minimum version for 6.8.x tested with the operator
+	MinVersion68x = "6.8.5"
+	// Minimum version for 7.x tested with the operator
+	MinVersion7x = "7.1.1"
+	// Current latest version for 7.x
+	LatestVersion7x = "7.6.0" // version to synchronize with the latest release of the Elastic Stack
 )
 
-// MinVersionOrSkip run this test only if the server has at least version v.
-func MinVersionOrSkip(t *testing.T, v string) {
-	info, err := ServerVersion()
-	require.NoError(t, err)
+func SkipIfMinVersion68x(t *testing.T) {
```

Comment:
May be we can have a generic function instead?

```go
func SkipIfStackVersionIsBetween(t *testing.T, minVersion, maxVersion string) {
   // code to skip if the minVersion <= test Stack version <= maxVersion 
}
```

Reply:
Yes, let's try to do something more generic. However, I don't feel that using an interval of version will be the most easy way to express how to skip an invalid upgrade.